src/components/SettingsItem.tsx

```diff
@@ -158,6 +158,40 @@ export function SettingsItemInput({
   )
 }
 
+type SettingsItemCtaProps = {
+  ctaText?: string | null
+  ctaColor?: colors
+  showChevron?: boolean
+} & BaseProps
+
+export function SettingsItemCta({
```

Comment:
I know this is a minor component, but could this use a test or two? Looks like the other components exported from this file have tests. I suppose this is already tested indirectly via the `Settings.test.tsx` file, however.

Reply:
added